src/compat/libc/stdlib/wcstombs.c

```diff
@@ -0,0 +1,34 @@
+/**
+ * @file
+ * @brief wcstombs function.
+ *
+ * @see stdlib.h
+ *
+ * @date 01.11.18
+ * @author Chubukov Filipp
+ */
+
+#include <stdlib.h>
+#include <string.h>
+#include <defines/size_t.h>
+#include <defines/wchar_t.h>
+
+size_t wcstombs(char *mbstr,  const wchar_t *wcstr, size_t max) {
+	int i = 0;
+	if (wcstr == NULL || mbstr == NULL) {
+		return 0;
+	}
+
+	while (*wcstr) {
+		if (i >= max) {
+			return i;
+		}
+	
+		*mbstr++ = *wcstr++;
+		i++;
+	}
+	if (*wcstr == '\0') {
```

Comment:
It seems that this condition is always true.

Reply:
u are right, reworked this place in wcstombs and mbstowcs